### ISSUE TITLE
v0.4.4 Server Expression API uses reserved python keywords

### ISSUE THREAD

**Issue by Steve-Fry** [krpc/krpc#440]
## Snippet
```
import krpc

conn = krpc.connect()
vessel = conn.space_center.active_vessel

alt_call = conn.get_call(getattr, vessel.flight(), "surface_altitude")
vspeed_call = conn.get_call(getattr, vessel.flight(), "vertical_speed")

alt_lt_exp = conn.krpc.Expression.less_than(
    conn.krpc.Expression.call(alt_call),
    conn.krpc.Expression.constant_double(5000)
)
vspeed_lt_exp = conn.krpc.Expression.less_than(
    conn.krpc.Expression.call(vspeed_call),
    conn.krpc.Expression.constant_double(450)
)
speed_and_alt_exp = conn.krpc.Expression.and(
    vspeed_lt_exp,
    alt_lt_exp
)
event = conn.krpc.add_event(speed_and_alt_exp)
with event.condition:
    event.wait()
```

## Expected Behaviour
Blocks until;
- vessel_altitude < 5000m 
- AND vertical_speed < 450

## Actual Behavior
```
  File "d:\DEV\Git\KSP\Missions\Test.py", line 17
    speed_and_alt_exp = conn.krpc.Expression.and(
                                              ^
SyntaxError: invalid syntax
```

## Versions
kRPC server
![image](https://user-images.githubusercontent.com/29991066/35484377-0bce32fa-0447-11e8-8836-085ae9f061e7.png)

kRPC client
![image](https://user-images.githubusercontent.com/29991066/35484415-722172a6-0447-11e8-925a-dccdf6fbaabb.png)

Python 
`Python 3.6.2 |Anaconda custom (64-bit)| (default, Sep 19 2017, 08:03:39) [MSC v.1900 64 bit (AMD64)]`

OS
Windows 10 - fully updated

## Detail
Python keywords can be tested using 
https://docs.python.org/3/library/keyword.html#keyword.iskeyword

To test those against the current list of available expressions (found at https://krpc.github.io/krpc/python/api/krpc/expressions.html) you can run the following;
```
import requests
import re
import keyword

cache_file = "source.html"

url = "https://krpc.github.io/krpc/_sources/python/api/krpc/expressions.rst.txt"
page_source = requests.get(url).content

str_page_source = page_source.decode("utf-8")

pattern = r"staticmethod:: (.*)\("
matches = re.findall(pattern, str_page_source)

for fn_name in matches:
    is_kw = keyword.iskeyword(fn_name)
    print("{:25} {}".format(fn_name, is_kw))
```

Bottom line these are the three that will not work under python (or at least in the way I have been calling them)
- `and`
- `or`
- `not`

**Comment by djungelorm** [comment]
Thanks for the detailed bug report.

These three methods are renamed dynamically by the python client to end with an underscore (e.g. ```conn.krpc.Expression.and_```). The implementation of this is here: https://github.com/krpc/krpc/blob/master/client/python/krpc/service.py#L32

This is a documentation issue - the documentation for the python client incorrectly claims that they are named without underscores. I'll fix this shortly.

**Comment by Steve-Fry** [comment]
Ahh, that is great news 👍 

I have to confess I didn't manage to make sense of the client. I need to spend a bit more time to understand it. 

Many thanks

**Comment by djungelorm** [comment]
Turns out this is a bug with the client AND the docs. I misread the code, and didn't test it properly... sorry!

I've just released python client version 0.4.4.post1 on pypi which should fix this. (The issue was that method names weren't being modified to avoid keyword clashes, only method parameter names were being renamed)
